tests/pio_tests/scenarios/basic_app_usecases.py

```diff
@@ -61,17 +61,13 @@ class BasicAppUsecases(BaseTestCase):
   def setUp(self):
     random.seed(3)
     self.log.info("Setting up the engine")
-
-    template_path = pjoin(
-        self.test_context.engine_directory, "recommendation-engine")
+    template_path = "https://github.com/chanlee514/template-scala-parallel-recommendation"
```

Comment:
That's a good idea. In fact, the templates have been donated and are in the process of migrating to Apache. I will ping you when that's done, so you can start using those new Git URLs as well.